lib/systemu.rb

```diff
@@ -94,7 +100,7 @@ def systemu
                   raise unless Exception === e
                   raise e
                 rescue
-                  raise "systemu: Error - process interrupted!\n#{ buf }\n"
+                  raise Error "systemu: Error - process interrupted!\n#{ buf }\n"
```

Comment:
This should be ```raise Error, "systemu: Error - process interrupted!\n#{ buf }\n"```which probably means this path is not covered in the tests.

Reply:
Good catch, sorry about that (fixed it).  And you're correct that it doesn't seem to be covered by the tests.